libevm/VMSIMD.cpp

```diff
@@ -426,108 +426,107 @@ void VM::xpush(uint8_t _b)
 	}
 }
 
-void VM::xget(uint8_t _b, uint8_t _c)
+void VM::xget(uint8_t _src_type, uint8_t _idx_type)
```

Comment:
Naming: don't use underscores other than for prefix, should be `_srcType` & `_idxType`